QgisModelBaker/libqgsprojectgen/db_factory/pg_factory.py

```diff
@@ -78,16 +78,19 @@ def post_generate_project_validations(self, configuration):
 
         return result, message
 
-    def get_tool_version(self):
+    def get_tool_version(self, db_ili_version):
         """Returns ili2pg version.
 
         :return: str ili2pg version.
         """
-        return '4.2.0'
+        if db_ili_version == 3:
+            return '3.11.2'
```

Comment:
That looks like something that we are going to forget about, no?

Reply:
no, the return `4.2.0` is a leftover

Reply:
It would be great if we could return an exact version, but if it's not possible to read that directly from the database, this can be postponed until we need it.

Reply:
You mean according to the database reading the exact version it has been created with?

Reply:
It's done here: https://github.com/opengisch/QgisModelBaker/pull/348/files#diff-f343a1b8c8f173b6ad8e03d24b25f2baR311 Meanwhile I did not find a better solution to evaluate with what version the DB has been created.